app/javascript/api/adapters/listing/RentalListingAdapter.ts

```diff
@@ -37,7 +39,13 @@ const ListingAdapter: Adapter<RailsRentalListing, Listing> = (item: RailsRentalL
       fileId: item.imageURL,
     },
   ],
-  events: [],
+  events: [
```

Comment:
Add a comment in here to make it clear that it's a temporary workaround for the listing directory page. Something like:
```
// TODO: Solve how to clearly communicate about lotteries. This is a temporary solution to be able to display lottery results data on the listing directory page, but the lottery results date is not the same as the lottery start date.
```